st2api/st2api/app.py

```diff
@@ -60,7 +60,7 @@ def setup_app(config={}):
 
     spec = spec_loader.load_spec('st2common', 'openapi.yaml.j2')
     transforms = {
-        '^/api/v1/': ['/', '/v1/'],
+        '^/api/v1/': ['/', '/v1/', '/v1'],
```

Comment:
There was a small bug - we didn't correctly map `/v1/ to `/api/v1/` so hitting one of those paths would incorrectly return 404 instead of the root controller content.